src/main/kotlin/build/buf/gradle/ProtobufGradlePluginSupport.kt

```diff
@@ -96,21 +125,14 @@ private fun Task.workspaceCommonConfig() {
             .tasks
             .matching { it::class.java.name == "com.google.protobuf.gradle.ProtobufExtract_Decorated" },
     )
-    createsOutput()
 }
 
-private fun Task.workspaceSymLinkEntries() =
-    allProtoDirs()
-        .map { project.makeMangledRelativizedPathStr(it) }
-        .joinToString("\n") { "|  - $it" }
-
 // Returns all directories that have may have proto files relevant to processing the project's proto files. This
 // includes any proto files that are simply references (includes) as well as those that will be processed (code
 // generation or validation).
 private fun Task.allProtoDirs() =
     project.allProtoSourceSetDirs()
         .plus(project.file(Paths.get(BUILD_EXTRACTED_INCLUDE_PROTOS_MAIN)))
-        .filter { anyProtos(it) }
```

Comment:
Why are these filters removed? If I remember correctly, this would cause a buf invocation to fail if any source directories didn't contain proto files. I hope there's a test that would fail if the only change to `main` was the removal of these filters, but I can't remember.

I see the `anyProtos` function is called elsewhere. Am I just not seeing the whole picture where the filter is applied later?

Reply:
These calls are just moved around the code. See this commit (or all commits in the sequence). There should not be any behavior change. If so, then I made a mistake.

Reply:
I've verified that my hope is reality - there are tests that fail if either or both of these filter calls are removed.